.github/workflows/build.yaml

```
@@ -2,8 +2,8 @@ name: Build and push container image
 
 on:
   pull_request:
-    paths:
-      - 'binder/**.yml'
+  paths:
+    - 'binder/**.yml'
```

```suggestion
    paths:
      - 'binder/**.yml'
```
I think the indentation is required here (https://github.com/carbonplan/forest-offsets-fires/actions/workflows/build.yaml)